### DISCUSSION TITLE
Zooz Zen25 with S2 and firmware 2.10

### DISCUSSION THREAD

**Discussion by danielsza** [zwave-js/zwave-js#6708] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Zooz zen25 plugs ( I have multiple with the same issue)
when they are added with S2 and especially with firmware 2.10
they will go dead, plus they no longer respond to any button presses until they are unplugged and replugged

What did you expect to happen instead?
not to go dead

 add the zen25 with S2 and either wait for a bit
or toggle it on/off multiple times and it will go dead

Anything else we should know?
If S2 isn't used it works properly
also firmware 2.0 seems to work better

### Software versions

Driver (node-zwave-js): ...
12.4.4

Z-Wave JS UI: ...
3.4.1

using a Zooz 800 stick
I've tired both firmware 1.2 and downgrading to 1.1
1.1 seems better overall

### Device information

Manufacturer: ... Zooz
Model name: ... Zen25 
Node ID: ... Node 11, but there are multiple

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-03-28.log](https://github.com/zwave-js/node-zwave-js/files/14795961/zwavejs_2024-03-28.log)

**Comment by AlCalzone** [comment]
Some parts of the log look like connectivity problems which cause the ZEN25s to go into panic mode and flood the network trying to reach the controller.
Did you follow the recommendations regarding controller placement [here](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/connectivity-issues?id=general-troubleshooting)?

**Comment by danielsza** [comment]
Thanks for getting back so quickly. Yes I read that document before. I have the stick on a 3' usb extension.

I don't think it's a placement issue, only because I have some other zen25's that have had have the same problem and they are within 6' of the controller.  I've attached an image of the placement of the stick... although currently i have a zooz 800 stick, in the image it's a 500 stick.

I'm think of possibly trying to go back to the 500 stick until a new firmware is available from zooz
or possibly trying this https://tubeszb.com/product/z-wave-poe-kit/ and placing it somewhere more central 


![IMG_3844](https://github.com/zwave-js/node-zwave-js/assets/6549750/79f672d6-d066-4658-9b9d-9ac85df4748f)

**Reply by AlCalzone** [reply]
There's also a 700/800 series firmware update scheduled for mid April, which is supposed to fix the current lockup issues. Maybe that is also related - at least it seems that your end devices are not getting the responses they expect.

**Comment by danielsza** [comment, marked as answer]
I ended up purchasing a Razberry Pro 7 along with the tubesz z wave PoE kit, and moving the controller to a more central location. Since then I’ve no longer had these zen25’s go dead. Granted that Razberry pro has newer firmware than the zooz 800 stick. I still get the controller unable to transmit error, but far less often. Hopefully silabs fixes the issue soon. Although currently the zwave network is working 95% better then with the zooz 800 setup.
